### PR TITLE
update the public suffix list

### PR DESCRIPTION
**Pull request by jeffkaufman**

### CODE REVIEW

File: src/third_party/effective_tld_names/README.drp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 URL: http://publicsuffix.org/
      https://publicsuffix.org/list/effective_tld_names.dat
-Version: Downloaded 2/17/2015 from
+Version: Downloaded 1/12/2017 from
          https://publicsuffix.org/list/effective_tld_names.dat
 License: MPL 1.1
```
Comment (morlovich) [inline src/third_party/effective_tld_names/README.drp]: Note that this License: bit is no longer correct, IIRC -- I think it's MPL2 now? May be a good time to correct it

File: src/third_party/effective_tld_names/effective_tld_names.dat [hunk 2]
```diff
@@ -1,10 +1,15 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Please pull this list from, and only from https://publicsuffix.org/list/public_suffix_list.dat,
```
Comment (morlovich) [inline src/third_party/effective_tld_names/effective_tld_names.dat]: So the URIL here doesn't match one in our README?

File: src/third_party/effective_tld_names/README.drp (hunk 1 above)
Comment (jeffkaufman) [inline src/third_party/effective_tld_names/README.drp]: fixed: e857d9c

File: src/third_party/effective_tld_names/effective_tld_names.dat (hunk 2 above)
Comment (jeffkaufman) [inline src/third_party/effective_tld_names/effective_tld_names.dat]: Good catch!  Fixed: e857d9c

**Comment by morlovich** [comment]
LGTM

**Pull request merged**